packages/launch-config/src/launch-config-crud/create.ts

```diff
@@ -44,3 +51,127 @@ export async function createLaunchConfig(rootFolder: string, fioriOptions: Fiori
     }
     return fs;
 }
+
+/**
+ * Writes the application info settings to the appInfo.json file.
+ * Adds the specified path to the latestGeneratedFiles array.
+ *
+ * @param {string} path - The project file path to add.
+ * @param log - The logger instance.
+ */
+export function writeApplicationInfoSettings(path: string, log?: Logger): void {
+    const appInfoFilePath: string = getFioriToolsDirectory();
+    console.log('appInfoFilePath', appInfoFilePath);
+    const appInfoContents = fs.existsSync(appInfoFilePath)
+        ? JSON.parse(fs.readFileSync(appInfoFilePath, 'utf-8'))
+        : { latestGeneratedFiles: [] };
+    console.log('appInfoContents', appInfoContents);
```

Comment:
Will this print the JSON or just `object`?

Reply:
Thanks for pointing this unnecessary console log - have removed this.